cocotb/share/lib/fli/FliImpl.cpp

```diff
@@ -1057,7 +1058,19 @@ FliTimedCbHdl *FliTimerCache::get_timer(uint64_t time) {
     return hdl;
 }
 
-void FliTimerCache::put_timer(FliTimedCbHdl *hdl) { free_list.push(hdl); }
+#ifndef FLI_TIMER_CACHE_SIZE
+#define FLI_TIMER_CACHE_SIZE 256
+#endif
```

Comment:
Can we make this a variable instead of a define? And can you add a comment on its sizing? (something vague as "Arbitrary size, shown to be sufficient" would do the trick, unless you've thought about it more)